gravitee-apim-cypress/cypress/integration/bulk/cypress-config.json

```
@@ -1,6 +1,6 @@
 {
   "baseUrl": "http://localhost:8083",
-  "integrationFolder": "cypress/bulk",
+  "integrationFolder": "cypress/tests/bulk",
```

that's a reason why your tests fails -> this path leads to nowhere
```suggestion
  "integrationFolder": "cypress/integration/bulk",
```

---

Good catch, thanks!